src/pages-and-resources/pages/PageCard.jsx

```diff
@@ -48,7 +50,8 @@ function PageCard({ intl, page }) {
       >
         <div className="d-flex flex-row">
           <span className="font-weight-bold">{page.title}</span>
-          {page.showSettings && <FontAwesomeIcon icon={faCog} className="ml-auto" />}
+          {page.showSettings
+          && <FontAwesomeIcon icon={faCog} className="ml-auto" onClick={()=> dispatch(showSettingsModal(page.id))} />}
```

Comment:
I think this might also remove the need to put `settingsModalAppId` in redux, if PagesAndResources is able to just read the route's appId to know which modal to load and display.

Reply:
@davidjoy  Sure, I was thinking along those lines initially, but wasn't sure how a modal lined up with a page route, especially when you cancel, but I've since figured it out. I will update it to use routing instead.